### PR TITLE
Add manager information from node BMC 

### PR DESCRIPTION
**Pull request by davidallendj**
This PR adds additional information to the `collect` output, including the `MACAddr` and `Managers` properties. The `Managers` property includes information found via the `/redfish/v1/Managers` endpoint as well as the manager's ethernet interfaces.

### CODE REVIEW

File: internal/collect.go [hunk 1]
```diff
@@ -21,7 +22,9 @@ import (
 
 	"github.com/Cray-HPE/hms-xname/xnames"
 	_ "github.com/mattn/go-sqlite3"
+	"github.com/stmcginnis/gofish"
 	_ "github.com/stmcginnis/gofish"
```
[the remark this answers is not in the document]
Comment (davidallendj) [reply internal/collect.go:26]: I didn't even notice this here. One of them must have snuck in via `goimports` I suppose.

[the remark this answers is not in the document]
Comment (synackd) [reply internal/collect.go]: Resolved with [2650210](https://github.com/OpenCHAMI/magellan/pull/62/commits/265021056f9fa1b5d0a6985821b22b9abb30bf54).

**Comment by synackd** [comment]
@davidallendj Looks like this has conflicts that need to be resolved. Adding the "not ready for review" label again until this is fixed.

**Review by synackd — APPROVED** [review]
Looks good to me.

These changes caused me to revisit #59. We should, in the near future, address how we should name things in an easy but flexible way. It's out of scope for this PR, of course, but it would be quite useful in general.

**Pull request merged**